crates/rustc_codegen_spirv/src/codegen_cx/entry.rs

```diff
@@ -554,4 +554,38 @@ impl<'tcx> CodegenCx<'tcx> {
             }
         }
     }
+
+    fn location_size_of_type(&self, ty: Word) -> u32 {
```

Comment:
Can this be renamed to `location_count_of_type` or `location_slots_per_type`? "size_of" could get confused with the size in bytes, which AFAICT is not relevant here.